pages/dmp_supporting_info/support_04_store_protect.md

```diff
@@ -37,6 +37,8 @@ This chapters collects information about data storage and backup throughout the
 
 If specific solutions are chosen during data processing and analysis or preservation, this may be specified in the respective chapters.
 
+## Question-specific guidance
+
 ### Storage guidelines
```

Comment:
Should we indicate if a question is e.g. marked with HPC?

Reply:
Don't think so, we might not show the tags in the end(?)

Reply:
Might be confusing to users to see guidance on questions they cannot see?

Reply:
Are the colors for the tags fixed?
We could use the following:
![Static Badge](https://img.shields.io/badge/DSW-ELSI-%23D35400)
![Static Badge](https://img.shields.io/badge/DSW-HPC-%23CED4DA)
![Static Badge](https://img.shields.io/badge/DSW-Horizon%20Europe-%232ECC71)
![Static Badge](https://img.shields.io/badge/DSW-IT%20security%20Europe-%23BDC3C7)
![Static Badge](https://img.shields.io/badge/DSW-RCN%2FScience%20Europe%20security%20Europe-%232980B9)
![Static Badge](https://img.shields.io/badge/DSW-cross%20institutional%20project-%23C0392B)
![Static Badge](https://img.shields.io/badge/DSW-machine%20actionable-%239B59B6)
![Static Badge](https://img.shields.io/badge/DSW-minimum-%23F39C12)
![Static Badge](https://img.shields.io/badge/DSW-umbrella%20DMP-%233498DB)

Reply:
Only four tags with user relevance: SE, Minimum, ELSI, HPC? Colors are not fixed, yet.